.github/workflows/_deploy_kafka_test.yml

```diff
@@ -88,6 +88,31 @@ jobs:
         run: | 
           az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP  --overwrite-existing
           cd main/kafka
+
+          sudo cat /root/.docker/config.json 
+          sudo mv ~/.docker/config.json /root/.docker/config.json
+          echo "docker config.json looks like the following: "
+          sudo cat /root/.docker/config.json
+          curl -LO https://github.com/microsoft/kata-containers/releases/download/3.2.0.azl1.genpolicy0/genpolicy
+          curl -LO https://github.com/microsoft/kata-containers/releases/download/3.2.0.azl1.genpolicy0/genpolicy-settings.json
+          curl -LO https://github.com/microsoft/kata-containers/releases/download/3.2.0.azl1.genpolicy0/rules.rego
+          chmod +x genpolicy
+          chmod +x genpolicy-settings.json
+          chmod +x rules.rego
+          echo "Containerd Status"
+          systemctl status containerd
+          echo "Check toml File Status" 
+          sudo cat /etc/containerd/config.toml
+          echo "Repalce toml File Content"
+          sudo sed -i 's/disabled_plugins = \["cri"\]/disabled_plugins = \[\]/' /etc/containerd/config.toml
+          echo "Check toml File Status Again" 
+          sudo cat /etc/containerd/config.toml
+          echo "Restart Containerd"
+          sudo systemctl restart containerd
+          echo "Sleep for 60 Seconds"
+          sleep 60
```

Comment:
nit: after restart and sleep, there could be another check to see if `containerd` is running using `systemctl is-active`

Reply:
I'm a bit hesitant to use this because it does not work on my local but I'll give it a try.

Reply:
Gotcha, I think it returns 'active' or 'inactive', so doing some sort of check to see if it returns 'active' might be good
I tried locally with cron: `system is-active cron`